web-api/src/middleware/apiGatewayHelper.test.js

```diff
@@ -313,6 +289,18 @@ describe('getAuthHeader', () => {
     expect(response).toEqual(undefined);
     expect(error).toBeDefined();
   });
+
+  it('should not return the user token and should throw an error if token is not present in query or anywhere else', () => {
+    let error;
+    let response;
+    try {
+      response = getAuthHeader({});
+    } catch (err) {
+      error = err;
+    }
+    expect(response).toEqual(null);
+    expect(error).toBeUndefined();
```

Comment:
@ericsorenson I'm wondering if this should actually have a defined error in this case where the AuthHeader isn't passed - the test description indicates an error should be thrown.

Reply:
The test description was incorrect, we're updating it!

Reply:
Updated!